<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register2</class>
 <widget class="QDialog" name="register2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rejestracja</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rejestracja nowego użytkownika</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_firstName">
       <property name="text">
        <string>Imię:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_firstName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_lastName">
       <property name="text">
        <string>Nazwisko:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_lastName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_age">
       <property name="text">
        <string>Wiek:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_age">
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_email"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_login">
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_login"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Hasło:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_confirmPassword">
       <property name="text">
        <string>Potwierdź hasło:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_confirmPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Zarejestruj się</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
